pixel-saver@deadalnix.me/app_menu.js

```diff
@@ -37,7 +37,7 @@ function updateAppMenu() {
 	}
 	
 	LOG('Override title ' + title);
-	appMenu._label.set_text(title);
+	appMenu._label.text = title;
```

Comment:
with my version of GNOME Shell (3.16) updateAppMenu doesn't work because the set_text function doesn't exists, but is a property.
Just a few minutes ago I found this: https://github.com/emerinohdz/status-title-bar/pull/28/files.
So maybe with GNOME 3.16 the right function name is setText. In this moment I'm using another system, later I can try to see if setText exists.